### ISSUE TITLE
Trials/1.0.5 1.20.1 update forge version

### ISSUE THREAD

**Issue by makarasty** [IzzelAliz/Arclight#1420]
I am writing to request an update for the Arclight server core. Currently, our server is running on version Trials/1.0.5 with Minecraft 1.20.1. We would greatly appreciate it if you could update the core to be compatible with the latest version of Forge.

Updating the core will help improve the performance and stability of our server, as well as ensure compatibility with the latest mods and plugins. This is crucial for maintaining an optimal gaming experience for our community.

**Comment by Sr-Satan** [comment]
I beg for the same too, I would like to play again but my mods are not compatible, I need at least forge 3.22 :c

**Comment by makarasty** [comment]
> I beg for the same too, I would like to play again but my mods are not compatible, I need at least forge 3.22 :c

In principle, I figured out a bit how to change the Forge version myself using IntelliJ IDEA. I think others can do it too, although it doesn't always work as intended. There's also a point in compiling the code from https://github.com/IzzelAliz/Arclight/tree/Trials. I haven't checked the latest version myself, but they occasionally update the Forge version there

**Comment by qyl27** [comment]
Yes, we have no release, but unstable builds are still available, like https://github.com/IzzelAliz/Arclight/actions/runs/12783155019, you can download the nightly build,
